src/backend/distributed/sql/citus--10.2-4--11.0-1.sql

```diff
@@ -17,7 +17,12 @@
 #include "udfs/get_all_active_transactions/11.0-1.sql"
 #include "udfs/get_global_active_transactions/11.0-1.sql"
 
+#include "udfs/dump_local_blocked_processes/11.0-1.sql"
```

Comment:
does the dump prefix still make sense?

Reply:
Probably not necessary, though given these are mostly used for `citus_lock_waits` and not very much intended for calling regularly, I didn't give much thought about this.

So, I may change it to ` local_blocked_processes` /   `global_blocked_processes`. Does that sound better?

Reply:
Ok, given that these UDFs are low-level things that we only need for `citus_lock_waits` and isolation tests, I'm updating the names to: `citus_internal_local_blocked_processes` and `citus_internal_global_blocked_processes`.

We don't expect/want users to play with these low level UDFs as it contains the details that not many need (at all).